### PR TITLE
fix: output the general usage of cli

### LINKED ISSUE #1036: Help doesn't show all possible arguments
The binary can take a lot of arguments/env variables, but currently, the following is the output of the call to `parseable --help`
```sh
Parseable v1.6.3

Cloud Native, log analytics platform for modern applications.
Join the community at https://logg.ing/community.

Commands:
  local-store  Start Parseable with a drive as storage
  s3-store     Start Parseable with S3 or compatible as storage
  blob-store   Start Parseable with Azure Blob storage
  help         Print this message or the help of the given subcommand(s)

Options:
  -h, --help     Print help
  -V, --version  Print version
```

No listing of possible commandline arguments is shown as would be expected from a commandline utility.

### PR DESCRIPTION
**Pull request by TwistingTwists**
Fixes #1036 

### Description

Aims to make CLI more user friendly to use. This PR adds a hint text for indicating how to use API.

<hr>

This PR has:
- [ ] been tested to ensure log ingestion and log query works.
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [ ] added documentation for new or modified features or behaviors.

### CODE REVIEW

**Comment by TwistingTwists** [comment]
```bash
> ./target/debug/parseable
Parseable v1.6.3

Cloud Native, log analytics platform for modern applications.

Usage:
parseable [command] [options..]


Join the community at https://logg.ing/community.

Commands:
  local-store  Start Parseable with a drive as storage
  s3-store     Start Parseable with S3 or compatible as storage
  blob-store   Start Parseable with Azure Blob storage
  help         Print this message or the help of the given subcommand(s)

Options:
  -h, --help     Print help
  -V, --version  Print version


```

**Comment by TwistingTwists** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by TwistingTwists** [comment]
recheck

File: src/option.rs [hunk 1]
```diff
@@ -49,7 +49,12 @@ impl Config {
             .name("Parseable")
             .about(
                 r#"
-Cloud Native, log analytics platform for modern applications."#,
+Cloud Native, log analytics platform for modern applications.
+
+Usage: 
+parseable [command] [options..]
+
+"#,
```
Comment (nitisht) [inline src/option.rs:57]:
```suggestion

Help:
parseable [command] --help

"#,
```

**Review by nitisht — APPROVED** [review]

**Comment by nitisht** [comment]
Skipping CI because this is a text change only.

**Pull request merged**